setup.py

```diff
@@ -9,7 +9,8 @@
         'traitlets',
         'docker',
         'jinja2',
-        'prometheus_client'
+        'prometheus_client',
+        'recommonmark'
```

Comment:
also should we have a different requirements.txt inside the doc folder? binderhub doesn't require recommonmark to run, so am not sure if this belongs here.

What's the general convention for packages needed only for generating docs?

Reply:
I typically put packages needed for docs only in a requirements-doc.txt or environment.yml within the docs directory. See jupyterhub repo for more guidance.